source/quickstart.txt

```diff
@@ -73,6 +73,12 @@ Follow the instructions below to setup {+c2c-product-name+}.
       <create-new-cluster>` or to create a :ref:`self-managed cluster
       <server-replica-set-deploy>`.
 
+      .. note:: 
```

Comment:
Hi Ali, 

Due to the possible performance impact of this parameter, you might consider using the include in the behavior section of each of the ``connecting mongosync`` task pages, [atlas-to-atlas](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/connecting/atlas-to-atlas/), [onprem-to-onprem](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/connecting/onprem-to-onprem/), and [onprem-to-atlas](https://preview-mongodbajhuhmdb.gatsbyjs.io/cluster-sync/DOCS-15476-set-minsnapshothistorywindowseconds-c2c/connecting/onprem-to-atlas/). Also possibly on the [sharded page](https://preview-mongodbajhuhmdb.gatsbyjs.io/cluster-sync/DOCS-15476-set-minsnapshothistorywindowseconds-c2c/multiple-mongosyncs/) (if applicable, might wanna check in TR). 

something like:

```
Latency Considerations
~~~~~~~~~~~~~~~~~~

.. include:: /includes/fact-latency-considerations.rst

```

Reply:
Gotcha, thanks for directing me where to go!